### DISCUSSION TITLE
Which components can you add ScrollArea to directly as a prop?

### DISCUSSION THREAD

**Discussion by devstc** [mantinedev/mantine#1826] [Q&A] [answered]
I see in the docs for the _ScrollArea_ component under "Usage with other components" that you can do the following to add the _ScrollArea_ component directly to the _Navbar.Section_ component rather than having to wrap the _ScrollArea_ component around it:
`<Navbar.Section grow component={ScrollArea} mx="-xs" px="xs">`

I'm not sure if I'll get greater control over the scrollbar adding it this way, but I just found you could do this and it seemed pretty cool.
It mentions there are other components it can be added to as well, but only certain ones.  I searched, but I couldn't find a list anywhere that says which specific components those are.  Does such a list exist for what components the _ScrollArea_ component works with like this?

**Comment by rtivital** [comment, marked as answer]
You can use ScrollArea with any component that supports `component` prop. See documentation to find out which components are polymorphic.

**Reply by devstc** [reply]
Thanks Vitaly.  I found some info on polymorphic components under "Learn the Basics".  I didn't see a list, but I'll do some more exploring.
